build.gradle

```diff
@@ -62,38 +53,38 @@ sourceSets {
 	}
 }
 
-/**
-* Gradle uses this built-in function to locate declared
-* dependencies within Maven's central repositories.
-*/
+ /**
+ * Gradle uses this built-in function to locate declared
+ * dependencies within Maven's central repositories.
+ */
 
 repositories {
 	mavenCentral()
 }
 
-/**
-* Gradle grabs the JUnit dependencies from Maven central,
-* and grabs local dependencies from the project filetree.
-*/
+ /**
+ * Gradle grabs the JUnit dependencies from Maven central,
+ * and grabs local dependencies from the project filetree.
+ */
 
 dependencies {
 	testCompile 'junit:junit:4.12'
 	compile fileTree(dir: 'lib')
 	compile fileTree(dir: 'src')
 }
 
-/**
-* IgnoreFailures is set to true until the failure issues are resolved
-* (refer to github.com/rsanchez-wsu/RaiderPlanner/issues/21)
-*/
+ /**
```

Comment:
This slash in the first line of a multi-line comment does not get indented with a space.

Reply:
Finished & pushed